tests/TestCase.php

```diff
@@ -114,13 +115,25 @@ protected function clearDirectory($dirPath, $exceptPaths = []): void
         }
     }
 
-    protected function generateRequest($type, $uri, $data = [], $pathParams = [], $headers = [], $routeConditions = [], $controllerMethod = 'test'): Request
-    {
+    protected function generateRequest(
+        $type,
+        $uri,
+        $data = [],
+        $pathParams = [],
+        $headers = [],
+        $routeConditions = [],
+        $controllerMethod = 'test',
+        $isInvokeController = false,
```

Comment:
it's not good solution make it as argument, cause we already has the controllerMethod argument which will onflict with this one. Please implement separate method `generateInvokableRequest` simple helper